DataDefinitions/Vehicle.cs

```diff
@@ -10,24 +10,44 @@ public class Vehicle
         public string loadout { get; private set; }
         public int rebuilds { get; private set; }
 
+        public string vehicleDefinition
+        {
+            get => vehicleDef.edname;
```

Comment:
Needs null protection, as vehicleDef might be null.
`get => vehicleDef.edname;`

Reply:
Fixed.